labgrid/resource/ykushpowerport.py

```diff
@@ -15,3 +15,8 @@ class YKUSHPowerPort(Resource):
     serial = attr.ib(validator=attr.validators.instance_of(str))
     index = attr.ib(validator=attr.validators.instance_of(int),
                     converter=int)
+
+@target_factory.reg_resource
+@attr.s(eq=False)
+class NetworkYKUSHPowerPort(NetworkResource, YKUSHPowerPort):
```

Comment:
We don't use multiple inheritance for resources, so die users need explicitly choose which to support.

Reply:
Ack.